core/tests/protractor_desktop/topicAndStoryEditor.js

```diff
@@ -200,8 +184,8 @@ describe('Topic editor functionality', function() {
 
     topicEditorPage.get(topicId);
     topicEditorPage.moveToSubtopicsTab();
-    topicEditorPage.addSubtopic('Subtopic 1');
-    topicEditorPage.addSubtopic('Subtopic 2');
+    topicEditorPage.addSubtopic('Subtopic1');
```

Comment:
Why did this change?

Reply:
Yes, I agree that this can't be a cross-suite clash. It looks like the subtopic `Subtopic 1` is created here:

https://github.com/oppia/oppia/blob/0e57098315d9f97e32f9134868b2d2fa0d31a619/core/tests/protractor_desktop/topicAndStoryEditor.js#L76

and then renamed in the test you removed from line 87. Now that we're not renaming the subtopic, when we try to add a subtopic of name `Subtopic 1`, it fails because the subtopic already exists.

Does this diagnosis sound right to you? If so, then I agree that the long-term solution is to make the `it` blocks independent. I'm fine with putting this off for a future PR, but I'd prefer that we not rely on the subtle change in subtopic name from `Subtopic 1` to `Subtopic1`. Could you instead create a new topic for this `should assign a skill to, between, and from subtopics` block? That will make this block independent and shouldn't disturb any other tests since this is the last block.

Alternatively, we could just change the subtopic names to be more clearly distinct, but I'd prefer to leave this `it` block more robust to future problems like this if possible.